### PR TITLE
Update version identifiers

### PR DESCRIPTION
**Pull request by seanog8**
I identified more versions and added them that emba can identify them.

### CODE REVIEW

File: config/bin_version_strings.cfg [hunk 1]
```diff
@@ -373,6 +392,7 @@ smbd:strict:"^Version\ [0-9]\.[0-9]+\.[0-9]+[a-z]$"
 smbpasswd:strict:"Version\ [0-9]\.[0-9]+\.[0-9]+"
 smtpclient:binary:"This\ is\ SMTPclient\ Version\ [0-9]\.[0-9]+\.[0-9]+\ "
 snmpd::"NET-SNMP\ version:\ \ [0-9]\.[0-9]+\.[0-9]+"
+snort:strict:"Version [0-9](\.[0-9])*"
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:395]: snort:strict:"Version\ [0-9](\.[0-9])*"

File: config/bin_version_strings.cfg [hunk 2]
```diff
@@ -351,12 +368,14 @@ rdisc6::"ndisc6\:\ IPv6\ Neighbor\/Router\ Discovery\ userland\ tool\ [0-9]\.[0-
 rdnssd::"rdnssd\:\ IPv6\ Recursive\ DNS\ Server\ discovery\ Daemon\ [0-9]\.[0-9]\.[0-9]\ "
 Realtek_camera_tool::"Welcome\ to\ Realtek\ Camera\ Tool\.\ Version\ [0-9]\.[0-9]+\.[0-9]"
 ripd::"ripd\ version\ [0-9]\.[0-9]+\.[0-9]+"
+ripd::"^ripd\ version [0-9]\.[0-9]+a"
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:371]: ripd::"^ripd\ version\ [0-9]\.[0-9]+a"

File: config/bin_version_strings.cfg [hunk 3]
```diff
@@ -300,6 +315,7 @@ ntp::"ntp[a-z][a-z]\ -\ .*\ NTP\ query\ program\ -\ Ver\.\ [0-9]\.[0-9]\.[0-9]+[
 ntp::"ntp[a-z]\ -\ .*\ NTP\ query\ program\ -\ Ver\.\ [0-9]\.[0-9]+\.[0-9]+$"
 ntp::"ntp[a-z]\ -\ .*\ NTP\ query\ program\ -\ Ver\.\ [0-9]\.[0-9]\.[0-9]+[a-z][0-9]+$"
 ntp::"ntp[pq]\ [0-9]\.[0-9]\.[0-9]+p[0-9]+$"
+openser::"^version:\ openser [0-9]\.[0-9]\.[0-9]+\ \(arm\/linux\)$"
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:318]: would this also work?
openser::"^version:\ openser\ [0-9]\.[0-9]\.[0-9]+\ \([a-z]\/[a-z]+\)$"

File: config/bin_version_strings.cfg [hunk 4]
```diff
@@ -252,9 +264,12 @@ mke2fs::"mke2fs\ [0-9]\.[0-9]+\.[0-9]+"
 mkfs\.jffs[0-9]::"mkfs\.jffs[0-9]:\ error!:\ revision\ [0-9]\.[0-9]+$"
 mkfs.jffs2::"mkfs\.jffs2.*revision\ [0-9]\.[0-9]+$"
 mkfs.jffs2::"mkfs\.jffs2\ revision\ [0-9]\.[0-9]\.[0-9]+\.[0-9]+"
+mkfs.xfs::"^mkfs\.xfs\ version\ [0-9]+\.[0-9]+\.[0-9]+$"
+mklost_plus_found::"^mklost\+found\ [0-9]+\.[0-9]+\.[0-9]+ \(.*\)$"
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:268]: mklost_plus_found::"^mklost\+found\ [0-9]+\.[0-9]+\.[0-9]+\ \(.*\)$"

File: config/bin_version_strings.cfg [hunk 5]
```diff
@@ -108,6 +116,7 @@ flatfsd:binary:"flatfsd\ [0-9]\.[0-9]\.[0-9]mtd"
 forked_media_server::"Forked\ Media\ Server:\ Version\ [0-9]\.[0-9]+"
 freebsd:binary:"FreeBSD\ [0-9]+\.[0-9]-RELEASE-p[0-9]\ "
 freebsd:binary:"FreeBSD\ [0-9]+\.[0-9]-RELEASE\ "
+freeradius::"^radiusd:\ FreeRADIUS\ Version [0-9]\.[0-9]+\.[0-9]+"
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:119]: freeradius::"^radiusd:\ FreeRADIUS\ Version\ [0-9]\.[0-9]+\.[0-9]+"

File: config/bin_version_strings.cfg [hunk 6]
```diff
@@ -8,23 +12,27 @@ apt::"apt\ [0-9]\.[0-9]+\.[0-9]+\ "
...
 avahi-dnsconfd::"avahi-dnsconfd\ [0-9]\.[0-9]+\.[0-9]+$"
 avahi-publish::"avahi-publish\ [0-9]\.[0-9]+\.[0-9]+$"
 avahi-resolve::"avahi-resolve\ [0-9]\.[0-9]+\.[0-9]+$"
 bash:binary:"[Bb]ash.*version\ [0-9]\.[0-9]+\.[0-9]+"
+batctl::"^batctl\ 20[0-9]+\.[0-9]\.[0-0]$"
 bc::"bc\ [0-9]\.[0-9]+\.[0-9]+"
 bcrelay::"BCrelay\ v[0-9]\.[0-9]\.[0-9]"
 binutils::"\(GNU\ Binutils.*\)\ [0-9]\.[0-9]+$"
 binutils::"\(GNU\ Binutils\ for\ Debian\)\ [0-9]\.[0-9]+$"
+blkid::"^blkid [0-9]\.[0-9]\.[0-9]\ \(.*\)$"
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:28]: blkid::"^blkid\ [0-9]\.[0-9]\.[0-9]\ \(.*\)$"

File: config/bin_version_strings.cfg [hunk 7]
```diff
@@ -8,23 +12,27 @@ apt::"apt\ [0-9]\.[0-9]+\.[0-9]+\ "
 apt::"apt\ [0-9]+\.[0-9]+\.[0-9]+\.[0-9]+\ "
 apt::"apt-Version\ [0-9]\.[0-9]+\.[0-9]+\ "
 arptables::"arptables\ v[0-9]\.[0-9]\.[0-9]-[0-9]:\ "
+asterisk::"^Asterisk\ [0-9]+\.[0-9]\.[0-9]+(.[0-9])*, Copyright"
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:15]: asterisk::"^Asterisk\ [0-9]+\.[0-9]\.[0-9]+(.[0-9])*,\ Copyright"

File: config/bin_version_strings.cfg [hunk 8]
```diff
@@ -1,3 +1,7 @@
+aircrack-ng::"^\ \ Air[a-z]*-ng [0-9]\.[0-9]+\ -\ \(C\)\ "
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:1]: aircrack-ng::"^\ \ Air[a-z]*-ng\ [0-9]\.[0-9]+\ -\ \(C\)\ "

File: config/bin_version_strings.cfg [hunk 9]
```diff
@@ -1,3 +1,7 @@
...
+aircrack-ng::"^\ \ ivsTools [0-9]\.[0-9]+\ -\ \(C\)\ "
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:2]: same here

File: config/bin_version_strings.cfg [hunk 10]
```diff
@@ -1,3 +1,7 @@
...
+aircrack-ng::"^\ \ makeivs-ng [0-9]\.[0-9]+\ -\ \(C\)\ "
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:3]: same here

File: config/bin_version_strings.cfg [hunk 11]
```diff
@@ -1,3 +1,7 @@
...
+aircrack-ng::"^\ \ Packetforge-ng [0-9]\.[0-9]+\ -\ \(C\)\ "
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:4]: same here

**Review by m-1-k-3 — CHANGES_REQUESTED** [review]
looks good. could you include the modifications please ...

File: config/bin_version_strings.cfg (hunk 3 above)
Comment (seanog8) [reply config/bin_version_strings.cfg:318]: I think there is a `+` missing.
I added this: 
`openser::"^version:\ openser\ [0-9]\.[0-9]\.[0-9]+\ \([a-z]+\/[a-z]+\)$"`

**Review by m-1-k-3 — APPROVED** [review]

**Comment by m-1-k-3** [comment]
looks good to me.
Thx @seanog8

**Pull request merged**